api/src/database.ts

```diff
@@ -1,8 +1,11 @@
 import { PrismaClient, Prisma } from '@prisma/client';
 import { Level as PinoLevel } from 'pino';
 import { getChildLogger } from './logger';
-import { getUserCrud, getSessionCrud, getChallengeCrud } from './modules';
+import { getUserCrud, getSessionCrud } from './modules';
+import { getChallengeCrud } from 'powerhouse-switchboard-plugin-demo'
```

Comment:
This is exactly what I want to prevent: that packages export multiple different functions that are then have to be manually put together in the main app

But removing `Challenge` module should already decrease complexity for this PR. As importing module that will provide functionality to other modules is too much

Reply:
see the updated version